doc/ref/string.xml

```diff
@@ -234,6 +236,17 @@ such digits. The meaning is given in the following list
     in &GAP;.
 </Item>
```

Comment:
If you follow my suggestion, you could augment the octal digits part by adding the `(from <C>0</C> to <C>7</C>)` bit as explanantion for the "octal digits"